### PR TITLE
global@v5.6.0 approval process for CircleCI

### PR DESCRIPTION
**Pull request by matthewhardern**
Adds approval process to CircleCI.

### CODE REVIEW

**Review by minrwhite — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -486,7 +486,11 @@ workflows:
 
   build_and_test:
     jobs:
+      - approval:
+          type: approval
```
Comment (ashleynolan) [inline .circleci/config.yml:490]: Just to check – is this approval for any build (including PR branches before merge?)

Comment (matthewhardern) [reply .circleci/config.yml:490]: Yes this is approval for any build including PR's as we want to be able to run master without interruption, so we need to be able to control all PR's and there builds, while we wait on a licence.

**Review by ashleynolan — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Comment by DamianMullins** [comment]
@matthewhardern Do you have a guide anywhere on how this process will work following this being merged?

**Comment by matthewhardern** [comment]
> @matthewhardern Do you have a guide anywhere on how this process will work following this being merged?

I will be putting together a meeting where we will agree a process with everyone who regularly uses this repo. Until then I will approve the PR's based on priority order (Master always first and then PR's in time order)

**Pull request merged**